src/common/history.h

```diff
@@ -55,6 +55,7 @@ void dt_history_delete_on_selection();
 /** compress history stack */
 void dt_history_compress_on_selection();
 void dt_history_compress_on_image(int32_t imgid);
```

Comment:
I would rename this one:
dt_history_compress_on_image_and_reload() // intended to be used in the darkroom

Reply:
Sure